mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -1020,6 +1020,12 @@
     <string name="notification_pbm_action_delete_and_open">Delete and Open</string>
     <!-- Name of the "Powered by Fenix" notification channel. Displayed in the "App notifications" system settings for the app -->
     <string name="notification_powered_by_channel_name">Powered By</string>
+    <!-- Name of the marketing notification channel. Displayed in the "App notifications" system settings for the app -->
+    <string name="notification_marketing_channel_name">Marketing</string>
+    <!-- Text shown in the notification that pops up to remind the user to set fenix as default browser. -->
```

Comment:
@rocketsroger Could you please add to the comment there that the placeholder will be replaced by "Fenix"? This gives localizers context for placeholders. thanks!

Reply:
Change merged.  Thanks!